types/luks.nix

```diff
@@ -16,6 +16,11 @@
       description = "Path to the key for encryption";
       example = "/tmp/disk.key";
     };
+    enableInitrdUnlock = lib.mkOption {
```

Comment:
maybe we can call it just `initrdUnlock`?

Reply:
Done :)